src/cursor/cur_layered.c

```diff
@@ -1343,6 +1376,57 @@ __clayered_next_random(WT_CURSOR *cursor)
     API_END_RET(session, ret);
 }
 
+/*
+ * __clayered_modify --
+ *     WT_CURSOR->modify method for the layered cursor type. This function assumes the modify will
+ *     be done on the btree that we originally calculate the diff from. Currently, we only allow
+ *     writes to the stable table so the assumption holds. TODO: revisit this when we enable writing
+ *     to the ingest table.
```

Comment:
Wouldn't this also apply to the case where we failover between calculating the delta and applying it? If so, is that a problem now rather than later?

Reply:
In failover, we should rollback all the transactions. So I think this should be fine. But it will be a problem if we allow writes before we drain the ingest table.

Reply:
Cool, thanks. I'm happy to leave this as a TODO then.